### PR TITLE
Issue #14937: Migrated section 7.2 The summary fragment to chapter wise testing

### PR DESCRIPTION
**Pull request by Zopsss**
#14937

### CODE REVIEW

**Comment by Zopsss** [comment]
@romani can pls restart this failing CI: https://checkstyle.semaphoreci.com/jobs/52cd47c2-d18f-474c-9b76-3e37b9936457

we encountered this error in my other PR before. Don't know why this is happening

File: src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule72thesummaryfragment/InputIncorrectSummaryFragment.java [hunk 1]
```diff
@@ -4,17 +4,19 @@
...
  * is a simple Javadoc. Some javadoc.
  */
-class InputIncorrectSummaryJavaDocCheck {
+class InputIncorrectSummaryFragment {
 
 /*warn*//**
      * As of JDK 1.1, replaced by {@link #setBounds(int,int,int,int)}
      */
     void foo3() {}
+    // violation 4 lines above 'First sentence of Javadoc is missing an ending period.'
```
Comment (romani) [inline src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule72thesummaryfragment/InputIncorrectSummaryFragment.java:14]: Please do `violation below` for all, the closer comment the better

**Review by romani — CHANGES_REQUESTED** [review]
Item

File: src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule72thesummaryfragment/InputIncorrectSummaryFragment.java (hunk 1 above)
Comment (Zopsss) [reply src/it/resources/com/google/checkstyle/test/chapter7javadoc/rule72thesummaryfragment/InputIncorrectSummaryFragment.java:14]: done

**Review by romani — APPROVED** [review]
Ok to merge

**Review by rdiachenko — APPROVED** [review]
lgtm

**Pull request merged**